packages/viewer/public/global.css

```diff
@@ -30,6 +30,14 @@
   color: #fff;
 }
 
+.example {
```

Comment:
For `a11y` reasons, the examples should be clickable elements. This should preserve the previous styling.